assets/scss/1_base/2-base.scss

```diff
@@ -0,0 +1,37 @@
+$color-primary: #151515;
+$green: #92c064;
+$green-hover: #a9cc88;
```

Comment:
if hover colors are lightened up a bit. You can try using scss functions:

`lighten($green, 10%)` or `mix(#fff, $green, 10%)`